### PR TITLE
New form

### PR DESCRIPTION
**Pull request by Puffin247**

### CODE REVIEW

**Review by dellydel — CHANGES_REQUESTED** [review]
Looks good so far. For the gender radio button, make sure the labels are for "male" or "female". looks like all 3 just have the label of "gender". Make those updates and push again. Good work so far.

File: Sanya_form.HTML [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            minlength="10"/>
+        
+        <br/>
+        
+      <b><label>Last Name:</label></b>
+      <input id="last_name" type="text" placeholder="Name" />
+        
+        <br/>
+
+      <b><label>Date_of_Birth:</label></b>
```
Comment (dellydel) [inline Sanya_form.HTML:28]: No need for the underscore in the label text. The user of your website will be able to see the label, so use plain text:  "Date of Birth"

File: Sanya_form.HTML [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      <input type="date" />
+
+        <br/>
+        
+      <b><label>Email:</label></b>
+      <input type="email" placeholder="email"/>
+        
+        <br/>
+        
+      <b><label>Gender:</label></b>
```
Comment (dellydel) [inline Sanya_form.HTML:38]: Looks like this gender label is out of place. It should be next to the radio buttons. If you view the page in a browser, you will see that it is not in the right place

File: Sanya_form.HTML [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      <input id="male" type="radio" />
+
+        <br/>
+
+      <b><label>Female:</label></b>
+      <input id="female" type="radio" name="gender">
+
+        <br/>
+
+      <b><label>Other:</label></b>
```
Comment (dellydel) [inline Sanya_form.HTML:62]: This label seems to be out of place

File: Sanya_form.HTML [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      <input id="other" type="radio" name="gender">
+        
+        <br/></b></b>
+      
+      <input type="submit" type="color-green">
```
Comment (dellydel) [inline Sanya_form.HTML:67]: You can only have one "type" attribute. I think you are looking for the "style" attribute

**Review by dellydel — CHANGES_REQUESTED** [review]

File: Sanya_form.HTML [hunk 5]
```diff
@@ -34,21 +34,21 @@
       <input type="email" placeholder="email"/>
         
         <br/>
-        
-      <b><label>Gender:</label></b>
-
-        <br/>
 
       <b><label>Mobile:</label></b>
       <input type="mobile" />
```
Comment (dellydel) [inline Sanya_form.HTML:39]: "mobile" is not a type.  There is a type that you can use for telephone numbers though

File: Sanya_form.HTML [hunk 6]
```diff
@@ -34,21 +34,21 @@
...
       <input type="address" maxlength="20" />
 
         <br/>
 
+      <b><label>Gender:</label></b>
+
+        <br/>
+
       <b><label>Male:</label></b>
       <input id="male" type="radio" />
```
Comment (dellydel) [inline Sanya_form.HTML:53]: This male radio button is not working properly

File: Sanya_form.HTML [hunk 7]
```diff
@@ -64,7 +64,7 @@
         
         <br/></b></b>
```
Comment (dellydel) [inline Sanya_form.HTML:65]: There are closing bold tags, but no opening tags. The bold tags can probably be removed

**Review by dellydel — CHANGES_REQUESTED** [review]

File: Sanya_form.HTML [hunk 8]
```diff
@@ -20,47 +20,47 @@
...
+      <input id="female" type="radio" name="gender" >
 
         <br/>
 
-      <b><label>Other:</label></b>
-      <input id="other" type="radio" name="gender">
+      <label>Other:</label>
+      <input id="other" type="radio" name="gender" >
         
         <br/></b></b>
```
Comment (dellydel) [inline Sanya_form.HTML:65]: Excellent updates, there are just the two closing bold tags that can be removed here and then you are good to go!

**Review by dellydel — CHANGES_REQUESTED** [review]

**Review by dellydel — APPROVED** [review]
Excellent

**Comment by Puffin247** [comment]
Hi Del,

Why is my mering blocked?

**Comment by dellydel** [comment]
> Hi Del,
> 
> Why is my mering blocked?

Your merge is blocked because you don't have the latest code from main. Remember you always need to have the latest code before you merge. This prevents merge conflicts. Pull from main, then merge those changes into your branch. or you can click on the update branch button on this page.

**Pull request merged**